modules/foundational.cft.yaml

```diff
@@ -259,7 +259,7 @@ Resources:
 
 Outputs:
   ConfigPostureRoleARN:
-    Description: ARN of the config posture role
+    Description: ARN of the config for posture role
     Value:
```

Comment:
nit: maybe we can change this to something more readable like 'ARN of the role for config posture' ?

Reply:
thanks, done